### PR TITLE
LPS-184525 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LPS-184525

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/Audit.macro [hunk 1]
```diff
@@ -7,7 +7,7 @@ definition {
 
 		ClickNoError(
 			key_menuItem = "Export Audit Events",
-			locator1 = "MenuItem#ANY_MENU_ITEM");
+			locator1 = "MenuItem#ANY_MENU_ITEM_1");
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/macros/Audit.macro:10]: @melodywu Could you define a more specific name?

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/macros/Audit.macro (hunk 1 above)
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/macros/Audit.macro]: Ok and done

**Comment by melodywu** [comment]
ci:test:sf

**Review by gaborlovasdotcom — APPROVED** [review]

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**